Tests/SwiftParserTest/DeclarationTests.swift

```diff
@@ -602,7 +640,12 @@ final class DeclarationTests: XCTestCase {
       """,
       diagnostics: [
         DiagnosticSpec(message: "expected declaration after 'public' modifier", fixIts: ["insert declaration"])
-      ]
+      ],
+      fixedSource: """
+        struct a {
+          public  <#declaration#>
```

Comment:
There should only be a single space here.

Reply:
https://github.com/apple/swift-syntax/issues/1611